repo-client/src/main/java/io/lionweb/repoclient/api/RepositoryVersionToken.java

```diff
@@ -0,0 +1,28 @@
+package io.lionweb.repoclient.api;
+
+import java.util.Objects;
+
+/** Value distinguishing a version of a particular repository. */
+public class RepositoryVersionToken {
+  private String token;
```

Comment:
`final`? Or are we on a sufficiently modern version of Java to use `record`?

Reply:
Making the field final. And not, we do not have support for record. We are supporting Java 8, which is very old. At some point we may want to move to Java 11, but I think records came later than that